wflambda.go

```diff
@@ -60,10 +63,33 @@ func Wrapper(lambdaHandler interface{}) interface{} {
 	handlerType = reflect.TypeOf(lambdaHandler)
 	handlerValue = reflect.ValueOf(lambdaHandler)
 
+	csCounter = Float()
+	invocationsCounter = Float()
+	errCounter = Float()
+	durationGauge = Float()
+
+	dc := &wavefront.DirectConfiguration{
+		Server:               server,
+		Token:                authToken,
+		BatchSize:            10000,
+		MaxBufferSize:        50000,
+		FlushIntervalSeconds: 1,
+	}
+
+	sender, err = wavefront.NewDirectSender(dc)
+	if err != nil {
+		return lambdaErrorHandler(err)
+	}
```

Comment:
I think its ok to push this into reportMetrics() - since it will only be called once per lambda execution.  Initializing the wavefrontDirectSender here is probably not going to result in re-use of this since we only report once per lambda execution. We should also initialize this only when WAVEFFRONT_ENABLED = true.

Reply:
That makes total sense. `reportMetrics()` will only be called when WAVEFFRONT_ENABLED = true anyways so that is a really good idea